implementations/rust/ockam/ockam_command/src/run/parser/resources.rs

```diff
@@ -212,3 +225,22 @@ pub fn parse_cmd_from_args(cmd: &str, args: &[String]) -> Result<OckamSubcommand
 pub trait ArgsToCommands<T> {
     fn into_commands(self) -> Result<Vec<T>>;
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn resolve_env_vars() {
```

Comment:
Also should there be test coverage for this error case (where env var can't be found) here